### ISSUE TITLE
Add Java API for <pre>

### ISSUE THREAD

**Issue by Wnt** [vaadin/flow#5010]
Add Java API for `<pre>` into com.vaadin.flow.component.html.

E.g.

```java
@Tag("pre")
public class Pre extends HtmlContainer {

	public Pre() {
		super();
	}

	public Pre(Component... components) {
		super(components);
	}

	public Pre(String text) {
		super();
		setText(text);
	}

}
```

**Comment by pleku** [comment]
This should be quite straight forward thing to add to https://github.com/vaadin/flow/tree/master/flow-html-components/src/main/java/com/vaadin/flow/component/html
And e.g. `Paragraph` and `ParagraphTest` works as an example of what is needed.

**Comment by pleku** [comment]
Closed via #6725 & #6767
